source/game_sa/Camera.cpp

```diff
@@ -826,7 +826,23 @@ void CCamera::SetColVarsAimWeapon(int32 aimingType) {
 
 // 0x50CC50
 void CCamera::SetColVarsPed(ePedType pedType, int32 nCamPedZoom) {
-    plugin::Call<0x50CC50, ePedType, int32>(pedType, nCamPedZoom);
+    if (pedType != PED_TYPE_PLAYER1 && pedType != PED_TYPE_PLAYER2) {
+        return;
+    }
+
+    int32 camColVars = 0;
+
+    if (pedType == PED_TYPE_PLAYER2) {
+        camColVars = nCamPedZoom + 6;
+    } else if (pedType == PED_TYPE_PLAYER1) {
+        camColVars = nCamPedZoom + 3;
+    }
```

Comment:
I'd prefer to use a `switch case here`
```suggestion
    switch (pedType) {
    case PED_TYPE_PLAYER1:
        camColVars = nCamPedZoom + 3;
        break;
    case PED_TYPE_PLAYER2:
        camColVars = nCamPedZoom + 6;
        break;
    default:
        return;
    }
```